vulnerabilities/helpers.py

```diff
@@ -79,3 +80,21 @@ def create_etag(data_src, url, etag_key):
 
 
 is_cve = re.compile(r"CVE-\d+-\d+", re.IGNORECASE).match
+
+
+def requests_with_5xx_retry(max_retries=5, backoff_factor=0.5):
```

Comment:
You said there were 104 errors,  how are they addressed ?

Reply:
104 came after retrying with short delays for multiple times. I increased the `backoff_factor` to 1 in the redhat importer.